### DISCUSSION TITLE
这个项目rabbitmq主要用来做什么的

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#215] [02.提问]
这个项目rabbitmq主要用来做什么的，服务部署的时候<img width="292" alt="image" src="https://user-images.githubusercontent.com/6092525/200273612-cc9a7470-1c80-4a96-8887-40219f8a7005.png">
哪几个是可选安装的

**Comment by zuihou** [comment]
3、5 可选
